lib/smart_proxy_dhcp_bluecat/bluecat_api.rb

```diff
@@ -11,7 +11,7 @@ class << self
     attr_accessor :token
   end
 
-  def initialize(scheme, verify, host, parent_block, view_name, config_name, config_id, server_id, username, password)
+  def initialize(scheme:, verify:, host:, parent_block:, view_name:, config_name:, config_id:, server_id:, username:, password:)
```

Comment:
Looks like, this will not work with ruby 2.0:
https://thoughtbot.com/blog/ruby-2-keyword-arguments

Reply:
yeah... named arguments where suggested in the first place by @timogoebel in our hallway session last week. but since we must support ruby2.0 im already on it to change it back to positional arguments

Reply:
another possibility would be to change this to an options hash.